go.mod

```diff
@@ -34,7 +34,7 @@ require (
 	rsc.io/letsencrypt v0.0.3 // indirect
 	sigs.k8s.io/controller-runtime v0.6.3
 	sigs.k8s.io/controller-tools v0.3.0
-	sigs.k8s.io/kubebuilder/v2 v2.3.2-0.20201111001842-c158f4fa4207
+	sigs.k8s.io/kubebuilder/v2 v2.3.2-0.20201204182959-6b775b860148
```

Comment:
We need to add a fragment for: 
- upgrade controller-runtime from v0.6.3 to v0.6.4

Reply:
Done